example/ctc/ocr_predict.py

```diff
@@ -21,8 +21,6 @@
 from __future__ import print_function
 import sys, os
 curr_path = os.path.dirname(os.path.abspath(os.path.expanduser(__file__)))
-sys.path.append("../../amalgamation/python/")
-sys.path.append("../../python/")
 
 from mxnet_predict import Predictor
```

Comment:
After removing the amalgamation python path insert above, does this still work ?

Reply:
When I was trying out these tests, I had to manually comment out these lines.
The ocr_predict.py used to proceed correctly and was able to find libmxnet_amalgmation.so  but it used to fail in loading the dependent libraries of libmxnet_amalagmation.so.  This might be due to the error in building amalgmation library on my machine.

Reply:
The script has a dependency on `mxnet_predict` module which is in amalgamation/python.  Does it work when you remove the sys.path and still import Predictor ? Are you sure you are not setting PYTHONPATH elsewhere ? I doubt it works since this is not part of the shipped python modules.

Reply:
Yes mxnet_predict is not the part of standard distribution but if I move the library to the standard library location, the script can load it.
We can keep the sys.path for mxnet_predict to avoid confusion. Please advice.

Reply:
Can we use the module api for prediction instead of mxnet_predict. Example: https://mxnet.incubator.apache.org/tutorials/python/predict_image.html

Reply:
I am not very experienced with this code. It would take me some time to understand and implement your suggestion.